### DISCUSSION TITLE
Oprogramowanie księgowe Symfonia na Windows Virtual Desktop

### DISCUSSION THREAD

**Discussion by KlekotJ** [AzureCommunityPL/group#186] [Pomoc]
Witam
Czy wdrażał ktoś oprogramowanie Symfonia na WVD. Czy to w ogóle dobry pomysł czy lepiej zrobić klasyczny serwer z RDP.

**Comment by eklime** [comment]
A jaką masz licencję na tą Symfonię? Chcesz  klient-serwer czy standalone? Ilu użytkowników?

**Reply by KlekotJ** [reply]
Klient serwer.  4-5 użytkowników.
Wyobrażam sobie tak że jest jedna maszyna jako serwer na którym jest zainstalowany SQL i jedna może dwie maszyny do obsługi WVD. 
Nie wiem o co chodzi z pytaniem o licencję?

**Reply by eklime** [reply]
Licencja - jaka wersja Symfonii.
Dla 4-5 użytkowników WVD to chyba przerost formy nad treścią. Rozumiem, że zależy Ci na spójnej wersji klientów, stąd chęć centralizacji, tak?

**Reply by smereczynski** [reply]
Czy ja wiem czy przerost... w WVD masz windows 10 multisession. Nigdzie indziej nie ma. To dobra ocpja.

**Reply by eklime** [reply]
No chyba, że w tym scenariuszu. Niemniej @KlekotJ masz na to jakiś określony budżet?

**Comment by KlekotJ** [comment]
Poniżej 1500 PLN miesięcznie, bo tyle mnie to teraz kosztuje. 
Jak najlepiej zainstalować oprogramowanie klienckie na stacjach? Czy przygotowywać obraz czy dać Windows z galerii i potem doinstalować?

**Reply by smereczynski** [reply]
To niezły budżet. Powinno wyjść taniej. Będziesz potrzebował AD, więc musisz postawić kontroler domrny np. na maszynie typu B (tanio) lub bezobsługowo Azure AD DS za $100/mc. Dla 4-5 userów klient/server to chyba przerost formy nad treścią. Lepiej jeden multisession i masz wszystkich ogarniętych orzy maszynie która nie będzie kosztowała milionów monet. Klient/serwer może się opłacić, jeśli chcesz host sesyjny wyłączać na noc i weekendy (a baza nie za dobrze znosi włącczenia i wyłączenia - nie wiem, czy tak jest. Jeśli tak nie jesst, to tym bardziej nie ma sensu).
Obraz możesz przygotować startując sobie maszynę w Azure i wyklikując co trzeba w systemie. Tak będzie najszybciej.

**Reply by KlekotJ** [reply]
Jeżeli postawię kontroler domeny to czy da się go zsynchronizować do Azure AD?
Po co mi kontroler domeny?

**Reply by smereczynski** [reply]
Kontroler jest wymagany przez WVD. Jeśli będzie to maszyna wirtualna, to jak najbardziej może synchronizować się do AAD.

**Reply by KlekotJ** [reply]
Możesz podrzucić jakiegoś linka jak to zrobić,, bo właśnie tego szukałem, ale nigdzie nie mogę znaleźć.

**Reply by smereczynski** [reply]
Jak co zrobić?

**Reply by KlekotJ** [reply]
Jak dodać kontroler domeny do AzureAD (bez posiadania kontrolera on-premise).

**Reply by smereczynski** [reply]
Nie rozumiem pytania... nic nie dodajesz. Jak masz w on-prem, to też nic nie dodajesz. Po rpostu stawiasz sobie ktroler w Azure na maszynie wirtualnej i instalujesz ad connect. Nic więcej.

**Reply by KlekotJ** [reply]
Mam użytkowników w AzureAD. Stawiam kontroler domeny. Instaluję AD connect i użytkownicy ściągają mi się na kontroler domeny?

**Reply by smereczynski** [reply]
Nie. Jeśli chcesz, żeby źródlem tożsamości było AAD, to musisz użyć AAD DS.

**Reply by KlekotJ** [reply]
Mógłbym postawić kontroler domeny i wyjdzie to taniej. Tylko wydaj mi się, że można zsynchronizować użytkowników z kontrolera do AzureAD a jeżeli mam ich już na AzureAD to przenieść ich na kontroler się już nie da. A przynajmniej nie znalazłem takiej informacji.

**Reply by smereczynski** [reply]
Zsynchronizować wstecz się nie da (choć podobno ta opcja ma kiedyś wrócić). Możesz utworzyć użytkowników w AD, zainstalować AD Connect i "zmaczować" ich po UPN albo innym parametrze.
